modules/security-deploy-image-sources.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-deploy.adoc
+
+[id="security-deploy-image-sources_{context}"]
+= Controlling what image sources can be deployed
+
+It is important that the intended images are actually being deployed, that the
+images including the contained content
+are from trusted sources, and they have not been altered. Cryptographic signing
+provides this assurance. {product-title} enables cluster administrators to apply
+security policy that is broad or narrow, reflecting deployment environment and
+security requirements. Two parameters define this policy:
+
+- one or more registries (with optional project namespace)
+- trust type (accept, reject, or require public key(s))
+
+You can use these policy parameters to allow, deny, or require a trust
+relationship for entire registries, parts of registries, or individual
+images. Using trusted public keys, you can ensure that the source is
+cryptographically verified.
+The policy rules apply to nodes. Policy may be
+applied uniformly across all nodes or targeted for different node workloads (for
+example, build, zone, or environment).
+
+.Example image signature policy file
+[source,yaml]
+----
+{
+    "default": [{"type": "reject"}],
+    "transports": {
+        "docker": {
+            "access.redhat.com": [
+                {
+                    "type": "signedBy",
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+                }
+            ]
+        },
+        "atomic": {
+            "172.30.1.1:5000/openshift": [
+                {
+                    "type": "signedBy",
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+                }
+            ],
+            "172.30.1.1:5000/production": [
+                {
+                    "type": "signedBy",
+                    "keyType": "GPGKeys",
+                    "keyPath": "/etc/pki/example.com/pubkey"
+                }
+            ],
+            "172.30.1.1:5000": [{"type": "insecureAcceptAnything"}]
+        }
+    }
+}
+----
+
+The policy can be saved onto a node as *_/etc/containers/policy.json_*. This
+example enforces the following rules:
+
+. Require images from the Red Hat Registry (`registry.access.redhat.com`) to be
```

Comment:
This list isn't an ordered procedure, so these should be regular unordered bullets.

Reply:
Yeah. I got it now. Fixed.